defaults/main.yml

```diff
@@ -36,6 +36,12 @@ stackdriver_apache_servers:
 stackdriver_elasticsearch_enabled: no
 stackdriver_elasticsearch_url: http://localhost:9200/_nodes/_local/stats/
 
+### jvm
+
+stackdriver_jvm_enabled: no
+stackdriver_jvm_host: localhost
+stackdriver_jvm_port: 5000
```

Comment:
I don't see your point here,  if you look at the current file all other plugins have defaults in this file. It is the default/main.yml. These parameters are meant to be set by the user in their own site.yml. Also, there is a check for connectivity which will fail if the settings are not properly set.